### PR TITLE
Simplify two if (remove redundant code)

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Remove redundant code

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/completion.go [hunk 1]
```diff
@@ -117,10 +117,9 @@ func NewCmdCompletion(out io.Writer, boilerPlate string) *cobra.Command {
 
 // RunCompletion checks given arguments and executes command
 func RunCompletion(out io.Writer, boilerPlate string, cmd *cobra.Command, args []string) error {
-	if len(args) == 0 {
+	if lenth := len(args); lenth == 0 {
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/completion.go]: `lenth` -> `length`

Comment (xichengliudui) [reply cmd/kubeadm/app/cmd/completion.go]: Sorry, I missed a character, which has been modified. thanks.

**Review by yagonobre — COMMENTED** [review]
Thanks @xichengliudui 
/lgtm

**Review by rosti — APPROVED** [review]
Thanks @xichengliudui !
/approve

**Pull request merged**